package.json

```diff
@@ -29,48 +29,53 @@
   "scripts": {
     "build": "ember build",
     "changelog": "lerna-changelog",
-    "start": "ember server",
+    "lint:js": "eslint ./*.js addon addon-test-support app blueprints config lib server test-support tests",
+    "start": "ember serve",
     "test:node": "mocha node-tests",
-    "test:debug": "mocha debug node-tests",
-    "test": "mocha node-tests && ember test"
+    "test:node:debug": "mocha debug node-tests",
+    "test": "mocha node-tests && ember test",
+    "test:all": "mocha node-tests && ember try:each"
   },
   "dependencies": {
-    "amd-name-resolver": "0.0.7",
+    "amd-name-resolver": "1.2.0",
     "babel-plugin-debug-macros": "^0.1.11",
     "babel-plugin-ember-modules-api-polyfill": "^2.3.0",
     "babel-plugin-transform-es2015-modules-amd": "^6.24.0",
-    "babel-polyfill": "^6.16.0",
-    "babel-preset-env": "^1.5.1",
-    "broccoli-babel-transpiler": "^6.1.2",
-    "broccoli-debug": "^0.6.2",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "broccoli-babel-transpiler": "^6.3.0",
+    "broccoli-debug": "^0.6.4",
     "broccoli-funnel": "^2.0.0",
     "broccoli-source": "^1.1.0",
     "clone": "^2.0.0",
-    "ember-cli-version-checker": "^2.1.0",
-    "semver": "^5.4.1"
+    "ember-cli-version-checker": "^2.1.2",
+    "ember-source-channel-url": "^1.1.0",
```

Comment:
Isn't this dependency only used for development mode? Is it necessary to keep it under `dependencies` or could it be moved to `devDependencies`?

Reply:
oops, thanks.

Reply:
https://github.com/babel/ember-cli-babel/pull/221